data/db/mil-tamron.xml

```diff
@@ -1,6 +1,109 @@
 <!DOCTYPE lensdatabase SYSTEM "lensfun-database.dtd">
 <lensdatabase version="2">
 
+    <lens>
+        <maker>Tamron</maker>
+        <model>E 17-70mm F2.8 B070</model>
```

Comment:
Is this the name that appears in the exif? Googling for the lens suggests that maybe it should include
> 17-70mm F/2.8 Di III-A VC RXD

although what you have is consistent with the other lenses in this file, so I'm not sure.

Reply:
The name of the lens is: 17-70mm F/2.8 Di III-A VC RXD but the model name in the xml file is what the script filled in, which one should I use?

Reply:
exiftool DSC07275.ARW | grep "Lens Model"
Lens Model                      : E 17-70mm F2.8 B070

Reply:
Seems like this is correct as it is then.